### ISSUE TITLE
win_lineinfile changes permissions of target file on update

### ISSUE THREAD

**Issue by pkunszt** [ansible/ansible-modules-core#2236]
especially annoying when changing some entries in \windows\system32\drivers\etc\hosts
after win_lineinfile adds an entry, the hosts file loses the users and applications permissions and is therefore not readable also by the network components at all. resetting the permissions is not enough, the network has to be rebooted.

**Comment by robynbergeron** [comment]
[module: windows/win_lineinfile.py]

**Comment by robynbergeron** [comment]
@pkunszt -- Thanks for submitting this bug report. You might want to consider giving us the following additional information to help the module maintainer identify the issue: https://github.com/ansible/ansible/blob/devel/ISSUE_TEMPLATE.md

@brianlloyd -- thanks in advance for taking a look at this bug report and for your ongoing work in maintaining this module.

**Comment by dbpeng** [comment]
Got the same issue here.  
@pkunszt  Any idea to work around for now?

**Comment by dbpeng** [comment]
## My env

ansible 2.1.0 (devel c1cec64aa8)
control machine: centos 6.5
target machine: windows 2008

Used the script in my playbook

``` yaml
- name: Make sure service1.myhost.com in hosts
  win_lineinfile: dest=C:\Windows\System32\drivers\etc\hosts regexp=service1.myhost.com line="{{ service1_ip }} service1.myhost.com"
  tags:
   - update_host
```

**Comment by bsuh** [comment]
Seems fixed by 9fa758d54c917a587f00a30388ca0ded2b863f51

**Comment by ghost** [comment]
Has this been fixed? If so, what version of ansible is this fixed in?

**Comment by bsuh** [comment]
It's fixed in 2.0.1.0 1
